src/Umbraco.Web/Editors/EntityController.cs

```diff
@@ -144,6 +144,37 @@ public IEnumerable<int> GetPath(int id, UmbracoEntityTypes type)
             return foundContent.Path.Split(new[] {','}, StringSplitOptions.RemoveEmptyEntries).Select(int.Parse);
         }
 
+
+
+
+        public dynamic GetUrl(int id, UmbracoEntityTypes type)

```

Comment:
I have updated this @Shazwazza to move this task along.

I understand originally why a `string` was not the return type of this method, as when the JS resource uses or grabs the value from the server it returns the string wrapped in quotes.

Updated just now to use HttpResponseMessage & return a raw StringContent result which gives us more control overt what we send back down the wire.

If you can review this C# change - as I don't just want to approve my own code before this gets marked as fixed & merged in.